platform/ui/src/components/userPreferencesForm/UserPreferencesForm.js

```diff
@@ -63,30 +40,35 @@ function UserPreferencesForm({
   onSave,
   onResetToDefaults,
```

Comment:
onResetToDefaults is not being used inside the component, remove from here +  update proptypes.

Reply:
Instead of removing it, I am keeping the functionality for now, added at the end of `onResetPreferences` and making sure it exists and its a function.